### ISSUE TITLE
[User flows and navigating] Link or button is not working as expected or did not navigate to the intended destination (03.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28658]
#### VFS team name
Claims & Appeals

#### VFS product name
Higher Level Review

#### Point of Contact/Reviewers
Peggy Gannon

---

### Platform Issue
Link or button is not working as expected or did not navigate to the intended destination.

### Issue Details
"Send the completed form to the benefit office " link doesn't take users to the correct place

### Supporting Guidance
[Category 03, Issue 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

---

### Platform Recommendation
Fix URL to send users to correct location.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by Mottie** [comment]
This issue is related to the update of the accordions to now use the `<va-accordion>` web component. The previous accordion would expand and scroll to it's content when the URL anchor targeted it. I reported this problem to the design system team - https://dsva.slack.com/archives/CBU0KDSB1/p1628801920405500

**Comment by shiragoodman** [comment]
updated labels to be consistent with post-pilot process. Updated category and issue number to reflect correct VA.gov Experience Standard 03.11

**Comment by timwright12** [comment]
@Mottie Had to revert this PR because it was causing duplicate ID a11y errors in the output:

The revert PR https://github.com/department-of-veterans-affairs/content-build/pull/751

**Comment by Mottie** [comment]
Thanks @timwright12 I should have realized that. I've got a new PR open - https://github.com/department-of-veterans-affairs/content-build/pull/764

**Comment by Mottie** [comment]
@shiragoodman This has been fixed! You can check in the HLR wizard in staging - the link leads to https://staging.va.gov/decision-reviews/higher-level-review/#find-addresses-for-other-benef-8622

**Comment by shiragoodman** [comment]
thanks @Mottie! You are free to close this ticket when the fix has been validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please let us know!
